igxDrop - error TS2341: Property '_data' is private and only accessible within class 'IgxDropDirective'.

## Description  

`error TS2341: Property '_data' is private and only accessible within class 'IgxDropDirective'`.

 * igniteui-angular version: `11.0.9`
 * igniteui-angular version: `~11.0.6`
 * browser: Chrome Version 87.0.4280.88 (Official Build) (64-bit)

## Steps to reproduce  

1. Create new project with Angular Cli 11 (`npm install -g @angular/cli`)
2. `ng new my-app`
3. Flow [Row Reordering Demo](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-drag#row-reordering-demo) to setup droppable Grid. I also created [this repository](https://github.com/loclv/angular-igniteui-row-reordering-demo-11) from this steps.

## Result

Err: `src/app/grid/grid-row-reorder/grid-row-reorder.component.html`

`error TS2341: Property '_data' is private and only accessible within class 'IgxDropDirective'`.

## Expected result  

Drag and drop a column to another order.

![Screenshot from 2021-01-08 16-01-09](https://user-images.githubusercontent.com/6302717/103995353-f4f6f100-51ca-11eb-9b15-a851b1d7ffff.png)

--- a/projects/igniteui-angular/src/lib/directives/drag-drop/drag-drop.directive.ts
+++ b/projects/igniteui-angular/src/lib/directives/drag-drop/drag-drop.directive.ts
@@ -1327,6 +1327,8 @@ export class IgxDragDirective implements AfterContentInit, OnDestroy {
 })
 export class IgxDropDirective implements OnInit, OnDestroy {
 
+    private _data: any;
+
     /**
      * - Save data inside the `igxDrop` directive. This can be set when instancing `igxDrop` on an element.
      * ```html
@@ -1335,7 +1337,6 @@ export class IgxDropDirective implements OnInit, OnDestroy {
      * @memberof IgxDropDirective
      */
     @Input('igxDrop')
-    private _data: any;
     public get data(): any {
         return this._data;
     }

